decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -116,6 +116,13 @@ def maximum_votes_reached?
         votes.count >= maximum_votes
       end
 
+      # Public: Can accumulate more votres than maximum for this proposal.
+      #
+      # Returns true if can accumulate, false otherwise
+      def can_accumulate_more_than_maximum
```

Comment:
Should this end with `?`?

Reply:
I don't think so... Could you review it again, I changed some name methods. :)